src/library/scala/Option.scala

```diff
@@ -63,6 +63,25 @@ object Option {
  *  This allows for sophisticated chaining of $option values without
  *  having to check for the existence of a value.
  *
+ * These are useful methods that exist for both $some and $none.
+ *  - [[isDefined]] — True if not empty
+ *  - [[isEmpty]] — True if empty
+ *  - [[nonEmpty]] — True if not empty
+ *  - [[orElse]] — Evaluate and return alternate optional value if empty
+ *  - [[getOrElse]] — Evaluate and return alternate value if empty
+ *  - [[get]] — Return value, throw exception if empty
+ *  - [[fold]] —  Apply function on optional value, return default if empty
+ *  - [[map]] — Apply a function on the optional value
+ *  - [[flatMap]] — Same as map but function must return an optional value
+ *  - [[foreach]] — Apply a procedure on option value
+ *  - [[collect]] — Apply partial pattern match on optional value
+ *  - [[filter]] — An optional value satisfies predicate
+ *  - [[filterNot]] — An optional value doesn't satisfy predicate
+ *  - [[exists]] — Apply predicate on optional value, or false if empty
+ *  - [[forall]] — Apply predicate on optional value, or true if empty
+ *  - [[contains]] — Checks if value equals optional value, or false if empty
+ *  - [[toList]] — Unary list of optional value, otherwise the empty list
```

Comment:
How about adding `toVector`?
Maybe toVector / toList with a description like: `Unary collection of optional value, otherwise the empty collection`

`List` is the simplest immutable collection, but AFICT `Vector` is "go to immutable, indexed sequence".  Has better default performance characteristics (https://docs.scala-lang.org/overviews/collections/performance-characteristics.html). By adding `toList` would encourage people to use it as default, `not toVector`.

Reply:
We actually just removed `Either` from the list for brevity.  I'm well aware that `List` isn't as performant.  I'm not sure how common converting`Option`s to a a sequence is, though.   I added `toList` to show that converting to sequences is possible.

Maybe your points about favoring `Vector` should be added to the docs for `List` and/or `Vector`?

https://www.scala-lang.org/api/2.12.8/scala/collection/immutable/List.html
https://www.scala-lang.org/api/2.12.8/scala/collection/immutable/Vector.html